### DISCUSSION TITLE
Generate Allure Report from Existing .json File

### DISCUSSION THREAD

**Discussion by vivntng** [allure-framework/allure2#1457] [General Discussion]
Is it possible to generate an Allure report just from a .json test result file? I'm not using any of the listed frameworks on the documentation so I'm wondering if it is possible and if it is, could someone give me some pointers on how to go about this?

**Comment by cheshi-mantu** [comment]
@vivntng
Hi Vivian,
if you aren't using any of the test frameworks, where did you get the result file?

**Reply by Avaddon-tfa** [reply]
I believe the issue here is that the framework @vivntng uses is not supported by Allure Report. 

For such cases, we have a manual for creating one's integration for Report: 
https://docs.qameta.io/allure-report/plugins/overview

@vivntng please, let me know if that's not what you needed.

**Comment by KurilovDmitry** [comment]
Hello all!
I want to run several postman collections in parallel, and get test results in json. Then run a program which unites the results in one big json and convert in to allure report.
How can I do it?

**Reply by douhaolee** [reply]
the possible quick way for postman is to use junitxml file as source of allure report - @KurilovDmitry
